scripts/install-check-tools.sh

```diff
@@ -17,11 +17,14 @@
 echo "Install soci check tools"
 set -eux -o pipefail
 
-curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(go env GOPATH)/bin" v1.56.2
+golangci_lint_ver="1.56.2"
+curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(go env GOPATH)/bin" v"${golangci_lint_ver}"
+# TODO: Verify script integrity via checksums
 go install github.com/kunalkushwaha/ltag@v0.2.4
 go install github.com/vbatts/git-validation@v1.2.0
 
 scversion="v0.10.0"
 arch=$(uname -m)
 wget -qO- "https://github.com/koalaman/shellcheck/releases/download/${scversion?}/shellcheck-${scversion?}.linux.${arch}.tar.xz" | tar -xJv
+# TODO: Verify script integrity via checksums
```

Comment:
Hmm, maybe another idea to circumvent this is to run the Docker [container](https://hub.docker.com/r/koalaman/shellcheck/tags) instead. I'll try this in another thread we don't have to get it here.

Reply:
Can we generate per-architecture shasums ourselves and hardcode them in this file? At least we will know that the files don't change from what we expect

Reply:
Sure, did it for both of them.